packages/trace-viewer/src/ui/watchMode.tsx

```diff
@@ -618,7 +618,7 @@ const refreshRootSuite = (eraseResults: boolean): Promise<void> => {
     return;
   }
 
-  receiver?.dispatch(message);
+  receiver?.dispatch(message).catch(() => {});
```

Comment:
Why catch? I think it would be better to see errors in the console, like before.

Reply:
This is inline with line 602 above.